setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = podman
-version = 4.9.0-dev
+version = 4.9.0
 author = Brent Baude, Jhon Honce
 author_email = jhonce@redhat.com
```

Comment:
@umohnani8 maybe you should add / update with your name here. wdyt @jwhonce ?

Reply:
I will do it in the next update :)